src/components/modals/TransactionModal/TransactionModal.tsx

```diff
@@ -0,0 +1,231 @@
+            }}
+          >
+            <span className="text-medium white bold">
+              {transactionType}({interactionType})&nbsp;:&nbsp;
+              {state?.state.ticker}
+            </span>
+            <span className="flex faded text" style={{ alignItems: 'center' }}>
+              Contract ID:&nbsp;
+              <CopyTextButton
+                copyText={contractId ? contractId.toString() : ''}
+                displayText={`${contractId!
+                  .toString()
+                  .slice(0, isMobile ? 6 : 15)}...${contractId!
+                  .toString()
+                  .slice(isMobile ? -6 : -15)}`}
+                size={25}
+                wrapperStyle={{
+                  fontStyle: 'bold',
+                  justifyContent: 'center',
+                  alignItems: 'center',
+                }}
+                position="relative"
+              />
+            </span>
+            <button className="modal-close-button" onClick={() => showModal()}>
+              <CloseIcon
+                width="20px"
+                height={'20px'}
+                fill="var(--text-white)"
+              />
+            </button>
+          </div>
+          {/** modal body - condition render edit or transfer */}
+          {interactionType === ANT_INTERACTION_TYPES.TRANSFER ? (
+            <div
+              className="flex flex-column"
+              style={{
+                width: '90%',
+                height: '100%',
+                gap: '1em',
+              }}
+            >
+              {/* todo: show 'transfer between accounts' option */}
+
+              <ValidationInput
+                inputClassName="data-input center"
+                inputCustomStyle={
+                  isValidAddress && toAddress
+                    ? {
+                        border: '2px solid var(--success-green)',
+                      }
+                    : !isValidAddress && toAddress
+                    ? {
+                        border: '2px solid var(--error-red)',
+                      }
+                    : !isValidAddress && !toAddress && {}
+                }
+                wrapperClassName="flex flex-column center"
+                placeholder="Enter recipients address"
+                maxLength={43}
+                value={toAddress}
+                setValue={setToAddress}
+                setIsValid={(validity: boolean) => setIsValidAddress(validity)}
+                showValidationChecklist={true}
+                validationPredicates={{
+                  [VALIDATION_INPUT_TYPES.ARWEAVE_ID]: (id: string) =>
+                    new Promise((resolve, reject) => {
+                      const validation = isArweaveTransactionID(id);
+                      if (validation === true) {
+                        resolve(validation);
+                      } else {
+                        reject(validation);
+                      }
+                    }),
+                }}
```

Comment:
why are we not using the same function we used before in ant creation? `arweaveDataProvider.validateArweaveID`?

Reply:
Ah, so validateArweaveID checks for transaction headers. Technically this input should implement checking to see if the id is an address or contract. We have neither of those implemented  - except for the validateTransactionTags I suppose - so I set it to this validation for now. I can implement the validateArweaveAddress here if we want though, shouldnt take much work.

Reply:
i think we can start with `validateArweaveID` for now.

Reply:
Yup - problem is for this modal, it will return false because wallet ID's do not exist on the arweave.net/tx endpoint, but rather the wallet id exists on the /wallet endpoint. 
![image](https://user-images.githubusercontent.com/85306700/218822557-cfecb0e2-7b18-4d72-9b5a-0ea7085ce547.png)

Reply:
but, added